packages/demo/usecase-demo/src/modules/ActiveRentals.tsx

```diff
@@ -33,8 +37,31 @@ export function ActiveRentals() {
 
       {isSuccess && userRentals.length > 0 && (
         <div className="flex flex-col p-12">
-          <div className="text-xl font-extrabold">Your rented content 🥰</div>
+          <div className="flex items-center justify-between">
+            <div className="text-xl font-extrabold">Your rented content 🥰</div>
+            {userRentals?.length > 0 && (
+              <div>
+                <button
+                  className="group p-1 transition-transform active:scale-[0.9]"
+                  onClick={() => OnScrollLeft(rentedContent)}
+                >
+                  <div className="rounded-full bg-grey-700 p-2 transition-colors group-hover:bg-grey-500/40">
+                    <ArrowLeft size="18" />
+                  </div>
+                </button>
+                <button
+                  className="group ml-1 p-1 transition-transform active:scale-[0.9]"
+                  onClick={() => OnScrollRight(rentedContent)}
+                >
+                  <div className="rounded-full bg-grey-700 p-2 transition-colors group-hover:bg-grey-500/40">
+                    <ArrowRight size="18" />
+                  </div>
+                </button>
+              </div>
```

Comment:
This code seems to be duplicated 3 times (`ActiveRentals.tsx`, `ActiveSubscriptions.tsx`, `ContentOfTheWeek.tsx`)
Maybe we can find a way to make it as a reusable component?

Reply:
Nice suggestion, I'll do that in my next commit.